docs/architecture/transaction-lifecycle.md

```diff
@@ -45,7 +45,7 @@ The transaction's block will then be subject to [conflation](./sequencer/conflat
 
 Occasionally, a batch may only consist of one block, with no conflation having taken place. This occurs when chain activity levels are particularly high, and the block size is larger than normal.
 
-Amongst Ethereum's L2 networks, Linea is the only network that uses batch conflation.
+Among Ethereum's L2 networks, Linea is the only network that uses batch conflation.
```

Comment:
This does not need to be changed. This is dialectical preference, and we do not enforce American English, British English, or any other as a rule.